sections/dashboard/PossibleActions/Layouts/LayoutLayerTwo.tsx

```diff
@@ -34,6 +38,11 @@ const LayoutLayerTwo: FC = () => {
 
 	const { stakingRewards, tradingRewards } = useUserStakingData(walletAddress);
 	const { currentCRatio, targetCRatio } = useStakingCalculations();
+	const snx = synthetix({ networkId: NetworkId['Mainnet-Ovm'], useOvm: true });
```

Comment:
You should be able to grab snx from useConnector. It will be instantiated already.

Reply:
Fixed